src/main/scala/com/raphtory/core/components/partition/StreamWriter.scala

```diff
@@ -41,11 +44,16 @@ class StreamWriter(
 
   var cancelableConsumer: Option[Consumer[Array[Byte]]] = None
 
+  val ingestionTimer = PartitionTelemetry.totalTimeForIngestion
```

Comment:
What is the purpose of this timer? It's just going to report at the end of the job how long the stream ingestor run for?

Reply:
Yes, this reports the timing for which the ingestion runs